packages/app/src/components/nominationInfo/styles.module.css

```diff
@@ -77,3 +77,8 @@
     font-size: large;
   }
 }
+
+.green {
+  color: #128b3e;
+  font-weight: bold;
+}
```

Comment:
Just a quick perusal of the codebase suggests we use this class in a bunch of places, including most links. Is it worth either making this class more globally available or styling all links with these colors (not sure if that's what we want, just seemed possible from that quick perusal)?

Reply:
Okay, good point. I added it to App.css for future use.